flink-core/src/main/java/org/apache/flink/metrics/MetricRegistry.java

```diff
@@ -64,6 +66,15 @@ public MetricRegistry(Configuration config) {
 		}
 		this.scopeFormats = scopeFormats;
 
+		char delim;
+		try {
+			delim = config.getString(ConfigConstants.METRICS_SCOPE_DELIMITER, ".").charAt(0);
```

Comment:
why do you post the same comment twice?

There isn't a particular reason for using a char. ScopeFormat.concat has a var arg String argument, and as such couldn't differentiate between cases with or without a delimiter. So i opted for the simple approach of using a char instead of investing more time into a feature for something that IMO won't be used anyway.